projects/ngx-formentry/src/form-entry/expression-runner/expression-runner.ts

```diff
@@ -150,6 +150,20 @@ export class ExpressionRunner {
     }
   }
 
+  private setControlQuestion(control: AfeFormArray | AfeFormGroup | AfeFormControl,
+    form: Form, scope: any) {
+    if (
+      control &&
+      control.controlRelations &&
+      control.controlRelations.relations) {
+      control.controlRelations.relations.forEach((relation) => {
+        const related = relation.relatedTo as any;
+        const question = form.searchNodeByQuestionId(related.uuid)[0]?.question?.extras;
+        scope['question' + related.uuid] = question;
```

Comment:
@enyachoke 
Suggestion: `scope["FORM"][related.uuid] = question;`
With this approach, we can access underlying row question using FORM.<id>.<field> in .json file.